draft-ietf-ppm-dap.md

```diff
@@ -3452,16 +3445,16 @@ batch bucket identifiers for the batch interval is
 [
   struct {
     start = batch_interval.start,
-    duration = time_precision,
+    duration = 1,
```

Comment:
Check this set over carefully - I think the meaning is still here, but I'm not an expert!